.github/workflows/windows.yml

```diff
@@ -184,10 +184,11 @@ jobs:
           if-no-files-found: 'error'
 
   openvino_tokenizers_cpack:
-    name: OpenVINO tokenizers extension (BUILD_FAST_TOKENIZERS=${{ matrix.build_fast_tokenizers }}) cpack
+    name: OpenVINO tokenizers cpack (BUILD_FAST_TOKENIZERS=${{ matrix.build_fast_tokenizers }}, BUILD_TYPE=${{ matrix.build_type }})
     strategy:
       matrix:
-        build_fast_tokenizers: [ON, OFF]
+        build_fast_tokenizers: [ON]
+        build_type: [Release, Debug]
```

Comment:
maybe we need to build fast tokenizers only for Debug?

Reply:
we need to test the both build types, previously the windows  Release build has been broken too

Reply:
yes, both release and debug, but:
- release tests prebuilt fast tokenizers
- debug builds fast tokenizers from scratch


But as I see pip wheel tests prebuilt tokenizers case. Ok then